pyaedt/maxwell.py

```diff
@@ -1920,17 +1920,24 @@ def export_element_based_harmonic_force(
         self.odesign.ExportElementBasedHarmonicForce(output_directory, setup, freq_option, f1, f2)
         return output_directory
 
-    @pyaedt_function_handler
-    def edit_external_circuit(self, netlist_file_path, schematic_design_name):
+    @pyaedt_function_handler()
+    def edit_external_circuit(self, netlist_file_path, schematic_design_name, parameters=None):
         """
-        Edit the external circuit for the winding.
+        Edit the external circuit for the winding and allow edit circuit parameter.
 
         Parameters
         ----------
         netlist_file_path : str
             Circuit netlist file path.
```

Comment:
```suggestion
            Path to the circuit netlist file.
```
Make this description parallel to the format of the next one--especially since three adjectives before the noun makes it hard to understand.